DigitalLearningSolutions.Web/ViewModels/MyAccount/EditDetailsViewModel.cs

```diff
@@ -1,76 +1,76 @@
-namespace DigitalLearningSolutions.Web.ViewModels.MyAccount
```

Comment:
Did you make a change to this file? Or is this just a correction of line endings.

Reply:
Correction of mixed line endings

Reply:
All looks good to me then